packages/cuba-react-ui/src/ui/table/DataTableHelpers.tsx

```diff
@@ -417,6 +397,7 @@ export interface TableChangeDTO<E> {
 /**
  * When called from `Table`'s `onChange` callback this function will reload data collection taking into account antd `Table`'s filters, sorter and pagination
  *
+ * todo do we really need this helper method? May be it better to inline it in DataTable#onChange
```

Comment:
It might be useful as a convenience method when using custom filters with vanilla `Table`.